unicodecomplete.py

```diff
@@ -65,6 +65,9 @@ def can_convert(view):
                 if symbol is not None:
                     if symbol_by_name(symbol):
                         return True
+                    elif enabled('accept_prefixes'):
+                        if symbol_by_unique_prefix(symbol):
```

Comment:
If we have `accept_prefixes` enabled _and_ havent found current prefix, should we go to `elif` below?
Maybe here should be `elif enabled('accept_prefixes') and symbol_by_unique_prefix(symbol):`?